cmd/root.go

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Did you mean to commit this file?

Reply:
I wasn't testing cobra-cli :face_exhaling: